lib/docs/scrapers/godot.rb

```diff
@@ -28,18 +28,13 @@ class Godot < UrlScraper
       end
     end
 
-    version '3.1' do
-      self.release = '3.1.1'
-      self.base_url = "http://docs.godotengine.org/en/#{self.version}/"
-    end
-
-    version '3.0' do
-      self.release = '3.0.6'
+    version '3.2' do
+      self.release = '3.2.1'
```

Comment:
Godot 3.2.3 is the latest stable version now:

```suggestion
      self.release = '3.2.3'
```